apps/train_tiny_effvit/main.py

```
@@ -0,0 +1,1773 @@
+
+    Raises:
+        RuntimeError: If required environment variables are not set
+    """
+    tracking_uri = os.environ.get("MLFLOW_TRACKING_URI")
+    experiment_id = os.environ.get("MLFLOW_EXPERIMENT_ID")
+
+    if not tracking_uri:
+        raise RuntimeError(
+            "MLFLOW_TRACKING_URI environment variable is not set. "
+            "Please set it to your MLflow tracking server URI."
+        )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)